docs/import/add_disaster.js

```diff
@@ -120,13 +118,30 @@ function writeNewDisaster(disasterId, states) {
   });
   if (!added) $('#disaster').append(createOptionFrom(disasterId));
   $('#disaster').val(disasterId);
+  disableDisasterPicker(false);
```

Comment:
nit: can you rename this to enableDisasterPicker? Search internally for "Improving Readability With Positive Booleans"

Reply:
Done.